src/Artsy/Router/buildServerApp.tsx

```diff
@@ -36,7 +36,7 @@ export function buildServerApp(config: RouterConfig & { userAgent?: string }) {
     getRelayEnvironment: () => {
       const relayEnvironment =
         (config.context && config.context.relayEnvironment) ||
-        createRelaySSREnvironment({ user })
+        createRelaySSREnvironment({ user, userAgent: config.userAgent })
```

Comment:
So it’s Force that would set this on the server?

Reply:
It's set in this function: https://github.com/damassi/force/blob/4683a3e889529213dfa9c357495ac1a0b9cc886b/src/desktop/lib/buildServerAppContext.ts#L20

Reply:
Gotcha, thanks 👍